apps/election-manager/src/__snapshots__/App.test.tsx.snap

```diff
@@ -935,7 +935,7 @@ exports[`printing ballots, print report, and test decks 2`] = `
             <span
               class="sc-dQppl kMSBOR"
             >
-              0 ballots
+              39 ballots
```

Comment:
The previous code erroneously did not have a number of ballots for either/neither elections in the test deck tally report